theme/banner/_base.scss

```diff
@@ -26,7 +27,7 @@
 	margin: 0;
 
 	li {
-		border-left: 1px solid #777;
+		border-left: 1px solid $wb-lng-glb-mn--pnl-btn-border-left;
```

Comment:
Reference to ``$wb-lng-glb-mn--pnl-btn-border-left`` should be renamed to ``$wb-lng-glb-mn-pnl-btn-border-left``.